react-front-end/tsrc/search/SearchPage.tsx

```diff
@@ -556,6 +527,7 @@ const SearchPage = ({ updateTemplate }: TemplateUpdateProps) => {
       externalMimeTypes: isSelectionSessionOpen()
         ? searchPageOptions.externalMimeTypes
         : undefined,
+      rawMode: searchPageOptions.rawMode,
```

Comment:
I think this warrants a comment:

```suggestion
      // As per requirements for persistence of rawMode, it is _not_ reset for New Searches
      rawMode: searchPageOptions.rawMode,
```